src/System.Runtime/tests/System.Runtime.Tests.targets

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <Target Name="CopyFilesToNonTPADir" BeforeTargets="RunTestsForProject" >
```

Comment:
You shouldn't use a custom target for this. You should just use SupplementalTestData like https://github.com/dotnet/corefx/blob/2b91aa87e5d6f5fd0808e6217a2c1f40fd776c8a/src/System.IO.Compression/tests/Performance/System.IO.Compression.Performance.Tests.csproj#L36